docker/entrypoint.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```

Comment:
As far as I understand this script is a kind of translator between config.js and env vars to be provided to the agent when it runs inside docker container.

Why don't just inject the env vars directly (i.e. ENV)? I don't see the point in this extra complexity... but maybe I'm missing something :)

Reply:
Yes your understanding is right.

For script I followed the same pattern as suggested in Issue #366 and https://github.com/telefonicaid/iotagent-node-lib/pull/726#discussion_r242214046 for Cygnus and PostGres.